adapter/internal/discovery/xds/rate_limiter_cache_test.go

```diff
@@ -646,6 +651,7 @@ func TestAddSubscriptionLevelRateLimitPolicy(t *testing.T) {
 					},
 				},
 				Organization: "carbon.super",
+				StopOnQuotaReach: true,
```

Comment:
Shall we add another test for `StopOnQuotaReach: true` and burst control enabled?